apps/frontend/tailwind.config.ts

```diff
@@ -4,6 +4,9 @@ const config: Config = {
     content: ["./src/**/*.{html,js,jsx,ts,tsx,mdx}", "../../node_modules/@tremor/**/*.{js,ts,jsx,tsx}"],
     theme: {
         extend: {
+            fontFamily: {
+                inter: ["Inter"],
```

Comment:
```suggestion
                sans: ["Inter"],
```

@afolasope to customize the default font in tailwindcss, we update the `theme.fontFamily.sans` in our tailwind config file or add a custom `base` style that sets `font-family` in our `css` file e.g.
```css
@layer base {
  html {
    font-family: "Inter", system-ui, sans-serif;
  }
}
```

[more...](https://tailwindcss.com/docs/font-family#customizing-the-default-font)